packages/server/src/jsRunner/tests/jsRunner.spec.ts

```diff
@@ -24,18 +9,28 @@ const { runJsHelpersTests } = require("@budibase/string-templates/test/utils")
 import tk from "timekeeper"
 import { init } from ".."
 import TestConfiguration from "../../tests/utilities/TestConfiguration"
+import environment from "../../environment"
 
 tk.freeze("2021-01-21T12:00:00")
 
-describe("jsRunner", () => {
+describe.each([
+  ["vm", false],
+  ["isolated-vm", true],
+])("jsRunner (using %s)", (_, useIsolatedVM) => {
   const config = new TestConfiguration()
 
   beforeAll(async () => {
+    environment._set("ISOLATEDVM_JS_RUNNER", useIsolatedVM)
```

Comment:
This is going to leak out of these tests and into other tests. I recommend using `config.withEnv`, which makes sure to set values back to what they were prior to being overridden.

Reply:
Removed this.